frontend/src/entities/emr/detail/emr-clusters-detail.tsx

```diff
@@ -67,13 +66,12 @@ function EMRDetail () {
         }
     }, [dispatch, basePath, projectName, clusterId, clusterName]);
 
-    // Refresh data in the background to keep state fresh
+    // Refresh data in the background to keep state fresh. While waiting for the EMR cluster to be ready, refresh every 3 seconds
     const isBackgroundRefreshing = useBackgroundRefresh(async () => {
         await dispatch(getEMRCluster(clusterId!));
-    }, [dispatch]);
+    }, [dispatch], true, clusterLoading ? 3 : window.env.BACKGROUND_REFRESH_INTERVAL);
```

Comment:
Obviously this was on purpose but doesn't 3s seem a bit aggressive?

Reply:
It typically took 3-9 seconds in testing for it to succeed

Reply:
I won't block on this.